### ISSUE TITLE
pysep overwrite data directory

### ISSUE THREAD

**Issue by carltape** [uafgeoteach/GEOS626_seis#8]
When I change the name for overwrite_event_tag, I still see a non-run that shows "data directory already exists". But it does not. Maybe for starters can we add the name of the data directory within the output statement? Not sure if this is a pysep issue or not.
<img width="783" alt="Screen Shot 2023-03-03 at 6 00 40 AM" src="https://user-images.githubusercontent.com/4455456/222753603-53ae6d0a-8bd1-4cd3-a42c-11f7be88bb81.png">

**Comment by aakash10gupta** [comment]
@carltape, this was a quick way to ignore the error that pysep throws when the output directory exists and let the notebook execution ('run all') proceed to the next cell smoothly. The problem here is that the try except block is also catching other errors instead of only the one that occurs when the output directory exists. To catch only the particular error of interest pysep will need to raise a particular kind of error which I am not sure if it does at the moment (@bch0w ?).

Another way to move forward is to run our own check for the existence of the output directory before doing the data download using pysep. This approach will replace the try except block with an if else block.

For the time being, to see the message for the actual error that occurs within pysep, replace the whole try except block with just the following lines of code with no indent -
`ps = Pysep(**download)`
`ps.run()`

**Comment by bch0w** [comment]
You can use the `overwrite` parameter in PySEP to have it automatically overwrite any existing directory it finds.
https://github.com/adjtomo/pysep/blob/3f5d7ab12aee663abe907036ca6eb757879d3722/pysep/pysep.py#L347-L349

Side note: probably best to not use the general except in practice, as you pointed out that will catch any exceptions during the data download/processing.

A more specific try-except block that would be more useful for determining errors would be:

```python
try:
    ps = Pysep(**download)
    ps.run()
except Exception as e:
    print(e)  # this will tell you what the error was
```

Still not great practice as this is very general and it's best to know exactly what exceptions you're trying to catch. 
If you want to catch more specific things you can do:

```python
try:
    ps = Pysep(**download)
    ps.run()
except ValueError:
    print("something bad happened")
except Exception as e:
    print(e)  
```

**Comment by aakash10gupta** [comment]
Resolved in the commit [8b3f295].
